project_proposal/milestones.md

```diff
@@ -0,0 +1,43 @@
+# Milestones
+Milestones: What are the key milestones throughout the project? 
+
+
+| activities               | dates     | week |
+| --                       | --        | --   |
+| kickoff                  | 01-09-20  | 36   |
+| Group meeting            | 04-09-20  | 37   |
+| Group meeting            | 11-09-20  | 38   |
+| Group meeting            | 18-09-20  | 39   |
+| Group meeting            | 25-09-20  | 40   |
+| Team leader meeting      | 25-09-20  | 40   |
+| Project draft            | 02-10-20  | 40   |
+| Peer feedback start      | 05-10-20  | 41   |
+| Peer feedback due date   | 09-10-20  | 41   |
+| Group meeting            | 09-10-20  | 41   |
+| Final project foundation | 18-10-20  | 42   |
+| Group meeting            | 23-10-20  | 43   |
+| Team leader meeting      | 09-10-20  | 43   |
+| Project week             | $$-10-20  | 43   |
+| Group meeting            | 30-10-20  | 44   |
+| Code review              | 30-10-20  | 44   |
+| Group meeting            | 06-11-20  | 45   |
+|                          |           | 46   |
+| Group meeting            | 21-11-20  | 47   |
+|                          |           | 48   |
+| Group meeting            | 05-12-20  | 49   |
+|                          |           | 50   |
+| Group meeting            | 18-11-20  | 51   |
+| Final report             | 02-12-20  | 51   |
+
+
```

Comment:
I get the idea of having two separate tables, but I think it would be better to have one instead, kinda like we had last term. Maybe add a column more, with 'Investigate', 'Requirements', etc. and then add the "smaller" milestones that fit these categories.

Reply:
not sure what you mean?